python/paddle/tensor/linalg.py

```diff
@@ -1963,6 +1963,199 @@ def svd(x, full_matrices=False, name=None):
         return u, s, vh
 
 
+def pca_lowrank(x, q=None, center=True, niter=2, name=None):
+    r"""
+    Performs linear Principal Component Analysis (PCA) on a low-rank matrix, batches of such matrices, or sparse matrix.
+
+    Let :math:`X` be the input matrix or a batch of input matrices, the output should satisfies:
+
+    .. math::
+        X = U * diag(S) * V^(T)
```

Comment:
上标没有上全
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/3a945a09-d6a2-4d47-ab1b-b2ec5ac9a38f)

```suggestion
        X = U * diag(S) * V^{T}
```